### ISSUE TITLE
Is  the self.mean_square_loss variable a loss function?

### ISSUE THREAD

**Issue by uniqueluo** [cardwing/Codes-for-prioritized-experience-replay#2]
Referece the paper  " Improving DDPG via Prioritized Experience Replay" at Algorithm 1 
       Minimize the loss function to update critic network: L= 1/K sum(Wi*(TD-error)^2)

Why plus self.critic_reg_loss value? what the self.critic_reg_loss value represent?

**Comment by cardwing** [comment]
self.critic_reg_loss is just a regularizer to penalize big weights (also called weight decay).

**Comment by uniqueluo** [comment]
thanks!
